### ISSUE TITLE
[ML] Error - Unable to bracket left percentile

### ISSUE THREAD

**Issue by edsavage** [elastic/ml-cpp#273]
Found in build

```
./platform.orig/darwin-x86_64/bin/autodetect --version
Model State Version 34
Quantile State Version 3
autodetect (64 bit): Version 7.0.0-alpha1-SNAPSHOT (Build 5aafd3b0eb2dc3) Copyright (c) 2018 Elasticsearch BV
```

Using data-set Iowa-liquor (https://data.iowa.gov/Economy/Iowa-Liquor-Sales/m3tr-qhgy)

with job config

``` 
 "analysis_config": {
    "bucket_span": "12h",
    "detectors": [
      {
        "detector_description": "sum(SaleDollars) partitionfield=\"CategoryName.keyword\"",
        "function": "sum",
        "field_name": "SaleDollars",
        "partition_field_name": "CategoryName.keyword",
        "detector_index": 0
      }
    ],
    "influencers": [
      "CategoryName.keyword",
      "City.keyword"
    ]
  }
```

Errors spotted in log file

```
[2018-10-23T11:18:29,835][ERROR][o.e.x.m.j.p.l.CppLogMessageHandler] [Edwards-MacBook-Pro.local] [iowa-liquor-sum-dollars-by-category_name3] [autodetect/42794] [CMultimodalPriorUtils.h@236] Unable to bracket left percentile = -11.2898, (a,b) = (-1.79769e+308,-1.79769e+308), (f(a),f(b)) = (6.11612,6.11612)
[2018-10-23T11:18:29,835][ERROR][o.e.x.m.j.p.l.CppLogMessageHandler] [Edwards-MacBook-Pro.local] [iowa-liquor-sum-dollars-by-category_name3] [autodetect/42794] [CMultimodalPriorUtils.h@264] Unable to bracket right percentile = -1.25001e-05, (a,b) = (1.79769e+308,1.79769e+308), (f(a),f(b)) = (-0.00566741,-0.00566741)
[2018-10-23T11:18:29,835][ERROR][o.e.x.m.j.p.l.CppLogMessageHandler] [Edwards-MacBook-Pro.local] [iowa-liquor-sum-dollars-by-category_name3] [autodetect/42794] [CGammaRateConjugate.cc@1159] Failed to compute log likelihood (3.209573632812500e+04 1.002935273247093e+02 (219.791, 10.3761) (219.791, 32244.3, 9.31738e+06) 1.000000000000000e+00 0.000000000000000e+00)
[2018-10-23T11:18:29,835][ERROR][o.e.x.m.j.p.l.CppLogMessageHandler] [Edwards-MacBook-Pro.local] [iowa-liquor-sum-dollars-by-category_name3] [autodetect/42794] [CGammaRateConjugate.cc@1160] samples = [inf]
[2018-10-23T11:18:29,835][ERROR][o.e.x.m.j.p.l.CppLogMessageHandler] [Edwards-MacBook-Pro.local] [iowa-liquor-sum-dollars-by-category_name3] [autodetect/42794] [CGammaRateConjugate.cc@1161] weights = [[1, 1, 1, 1]]
```

**Comment by edsavage** [comment]
Potentially related to #155

**Comment by dolaru** [comment]
Same issue spotted in `6.x` on **OSX only**, for the `event_rate_gen_trend` job:

**Comment by tveasey** [comment]
The original problems reported here have been fixed, for example #1268. I think we can tentatively close off this issue and open a new issue if we get log errors reported again.
